src/wp-includes/formatting.php

```diff
@@ -2924,7 +3056,9 @@ function antispambot( $email_address, $hex_encoding = 0 ) {
 	for ( $i = 0, $len = strlen( $email_address ); $i < $len; $i++ ) {
 		$j = rand( 0, 1 + $hex_encoding );
 
-		if ( 0 === $j ) {
+		if ( ord( $email_address[ $i ] ) > 127 ) {
```

Comment:
It’s possible to debate the value of this function today, but doesn’t this change effectively disable it for non-ASCII characters?

If we make such a large change it would be best to have discussion about it and do it intentionally.

There are tools we can use to work through UTF-8 code points and not have to make this choice. See my comment above on the script detection.

Reply:
It does disable the function for such addresses, and that isn't a problem right now. 

ICANN gathers abuse data (primarily to forward  it to registrars in real time), I have access to the statistics, and unicode addresses aren't a factor now. There are reasons for that and I'd be glad to talk about it, but whatever the reasons, a function called 'antispambot' can relax about that sort of address. It needs to deal with the effects, not with the reasons.

Reply:
It has to be changed as part of this patch: I became aware of it because a unit test broke. A breaking unit test is a serious matter.

I too feel that its utiliy is difficult to see. If you don't want it, fine, fine! If you do want something like it, I'd like to contribute something later, without holding up this PR.